packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -38,6 +44,13 @@ class TickerMode extends StatefulWidget {
   /// they just don't call their callbacks. Time still elapses.
   final bool enabled;
 
+  /// If true, tickers in this subtree will request frames using
+  /// SchedulerBinding.scheduleForcedFrame while active.
+  ///
+  /// Use sparingly, as this can increase battery usage when frames would
+  /// otherwise be suppressed by lifecycle.
```

Comment:
I think it'd be better to slightly explain what forced frames are here by reusing the docs on [`SchedulerBinding.scheduleForcedFrame`](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/scheduleForcedFrame.html). 

Consider:

```suggestion
  /// If true, tickers in this subtree will force frames even if
  /// frames would normally not be scheduled (e.g. even if the
  /// device's screen is turned off).
  ///
  /// Use sparingly as this will cause significantly higher battery
  /// usage when the device should be idle.
```